packages/helm-charts/testnet/templates/validators.statefulset.yaml

```diff
@@ -181,6 +186,16 @@ spec:
 
           [[ "$IN_MEMORY_DISCOVERY_TABLE" == "true" ]] && ADDITIONAL_FLAGS="${ADDITIONAL_FLAGS} --use-in-memory-discovery-table"
 
+          # Use a different fee recipient address if option available
+          set +e
+          geth --help | grep -q 'tx-fee-recipient'
+          tx_fee_available=$?
+          set -e
+          if [ $tx_fee_available ] && [ -f /root/.celo/feeRecipientAddress ] ; then
```

Comment:
I think if the `if` above is unnecessary and were removed then the second condition in this if could also be removed.

Reply:
Actually, I see a bigger problem now. If you give these flags as well as `--etherbase`, you will get an error and the node won't run:
```
Fatal: `etherbase` and `miner.validator` flag should not be used together. `miner.validator` and `tx-fee-recipient` constitute both of `etherbase`' functions
```

There should be an if/else and the `--etherbase` flag should be in the else and removed from line 1224 below

Reply:
Also, should the condition be `$tx_fee_available -eq 0` instead of this?